config/GTNewHorizons/blocklimiter.cfg

```diff
@@ -7,10 +7,10 @@ debug {
 
 
 main {
-    # Define your Blocks here. Syntax is: [modID]:[BlockID];[DimID];... if you don't add a Dimension (e.g. minecraft:dirt) it will be denied in every dimension [default: ]
+	# Define your Blocks here. Syntax is: [modID]:[BlockID];[DimID];... if you don't add a Dimension (e.g. minecraft:dirt) it will be denied in every dimension [default: ]
     S:BlockList <
-        ExtraUtilities:enderThermicPump;0;
-        ExtraUtilities:enderQuarry;0;
+        ExtraUtilities:enderThermicPump;0;0;1;
+        ExtraUtilities:enderQuarry;0;0;1;
      >
```

Comment:
those two should be removed from the config, and server owners should put what they need here. There is no reason to block those in OW and in End if players are on SP

Reply:
so comment it out entirely and let server owners find it?

I assume in general we tune the pack for "Single player" and assume server operators know what to look for - do we have a "gotchas on upgrade" somewhere?